rest/model/subscriptions.go

```diff
@@ -124,5 +123,9 @@ func (s *APISubscription) ToService() (interface{}, error) {
 		out.RegexSelectors = append(out.RegexSelectors, newSelector)
 	}
 
+	if err = out.Filter.FromSelectors(append(out.Selectors, out.RegexSelectors...)); err != nil {
```

Comment:
GitHub doesn't let me comment on unchanged lines, but should the `Filter` be added to the `APISubscription` data model, even if the selectors are going to stay?

Reply:
I think it will be simpler to add the filter to the APISubscription after the migration. Let's defer it to the next PR.